src/main/java/core/basesyntax/fileoperations/FileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.fileoperations;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        try {
+            BufferedReader reader = new BufferedReader(new java.io.FileReader(filePath));
+            StringBuilder builder = new StringBuilder();
+            String line = reader.readLine();
+            while (line != null) {
+                lines.add(line);
+                line = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
+        }
```

Comment:
It's good that you're throwing a RuntimeException with an informative message in the catch block, adhering to the checklist. However, you should use try-with-resources to ensure that the BufferedReader is closed properly after use. This is important to prevent potential resource leaks.